Helpjuice/ckeditor5
# [Editor] [sc-1318] Embedding videos on editors isn't working

## Describe the changes in this pull request

The current version doesn't add the protocol for videos on prod.

This PR changes the embed configuration to include it.

More information about this change:

- https://ckeditor.com/docs/ckeditor5/latest/features/media-embed.html#media-providers
- https://ckeditor.com/docs/ckeditor5/latest/api/module_media-embed_mediaembed-MediaEmbedProvider.html

### Before & After screenshots (if this pull request changes the UI)

I tried to test it using the sample editor in this repo, but it doesn't support it since it's an external media. You will have to test it in the app repo on this PR here: https://github.com/hajrice/helpjuice4/pull/2477

### Does this resolve an existing issue?

[sc-1318]

### How should this be tested during the review?

Check this PR here for testing it in the app: https://github.com/hajrice/helpjuice4/pull/2477
